src/main/java/ch/njol/skript/events/EvtEntityBlockChange.java

```diff
@@ -39,28 +39,30 @@ public class EvtEntityBlockChange extends SkriptEvent {
 	
 	static {
 		Skript.registerEvent("Enderman/Sheep/Silverfish/Falling Block", EvtEntityBlockChange.class, EntityChangeBlockEvent.class, ChangeEvent.patterns)
-				.description("Called when an enderman places or picks up a block, a sheep eats grass, ",
-						"a silverfish boops into/out of a block or a falling block lands and turns into a block respectively.")
-				.examples("on sheep eat:",
-						"\tkill entity",
-						"\tbroadcast \"A sheep stole some grass!\"",
-						"on falling block land:",
-						"\tif event-entity is a falling dirt:",
-						"\t\tcancel event")
-				.since("<i>unknown</i>, 2.5.2 (falling block)");
+			.description(
+				"Called when an enderman places or picks up a block, a sheep eats grass, ",
+				"a silverfish boops into/out of a block or a falling block lands and turns into a block respectively."
+			).examples(
+				"on sheep eat:",
+				"\tkill entity",
+				"\tbroadcast \"A sheep stole some grass!\"",
+				"on falling block land:",
+				"\tif event-entity is a falling dirt:",
+				"\t\tcancel event"
+			).since("<i>unknown</i>, 2.5.2 (falling block)");
 	}
 	
 	private enum ChangeEvent {
 
-		ENDERMAN_PLACE("enderman place", e -> e.getEntity() instanceof Enderman && e.getTo() != Material.AIR),
-		ENDERMAN_PICKUP("enderman pickup", e -> e.getEntity() instanceof Enderman && e.getTo() == Material.AIR),
+		ENDERMAN_PLACE("enderman place", event -> event.getEntity() instanceof Enderman && event.getTo() != Material.AIR),
```

Comment:
Is there a reason for this not using `ItemUtils#isAir` as well?

Reply:
Should I change those altogether?